vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -293,6 +293,7 @@ class RoomDetailFragment @Inject constructor(
         setupJumpToBottomView()
         setupWidgetsBannerView()
 
+        roomToolbarContentView.isClickable = false
```

Comment:
Useless, I think next line will restore isClickable to true

Reply:
Yes nice catch